main.tf

```diff
@@ -1,42 +1,22 @@
 terraform {
-  required_version = ">= 0.14.11"
+  required_version = ">= 0.12.17"
   required_providers {
     aws = ">= 3.0"
   }
 }
 
-locals {
-  filename = "${path.module}/lambda/function.zip"
+module "acs" {
+  source            = "github.com/byu-oit/terraform-aws-acs-info?ref=v3.5.0"
+  vpc_vpn_to_campus = true
```

Comment:
I thought we were trying to get away from our terraform modules using other external modules. Is there a reason you put the acs module back inside here?

Reply:
Just to simplify the work for developers. What I could do instead is remove ACS and put all these defaults in the readme example instead. What do you think?

Reply:
Yeah, that's a good recommendation to use the standard values in the readme. Most of our terraform modules do this (I think) including this one. I want to say that Hashicorp advised us to not embed modules inside modules because it can do some odd things (with path names of resources) especially if you change things within modules etc.